src/app/cluster/cluster-details/template.html

```diff
@@ -233,7 +243,7 @@
                   [isClusterRunning]="isClusterAPIRunning"></km-node-list>
   </div>
 
-  <km-tab-card>
+  <km-tab-card [ngClass]="{'hideMLA': !isMLAEnabled()}">
```

Comment:
The check if MLA is enabled need to wait for the seedSettings endpoint. This is causing problems, because it takes to long and the `*ngIf` on the `km-tab` will not work in this case because is doesn't refresh on it's own. (This btw is also a problem for other tabs, e.g. if OPA gets enabled via the edit cluster dialog, you manually need to refresh the page - just noticed now)
So, this little hack seems to be working, though with a slow delay. 
However, we should check if we could find a better solution for it (aka all tabs), but maybe in a seperate issue.

Reply:
You might try running the code from `TabCardComponent.ngAfterContentInit` (`this.tabs = this.inputTabs.toArray();`) in on changes event. This should fix the issue.

Reply:
There is definitely a way to handle this angular way. If Marcin's example won't work we will take a deeper look to try and avoid such workarounds.

Reply:
I might be doing something wrong, but I didn't manage to make it work. The on changes event has never been called for me 🤔

Reply:
This one https://angular.io/api/core/OnChanges?

Reply:
yes

Reply:
We'll take a look then

Reply:
okay, I guess I can open a new issue then?
Other than that I think this PR is ready to get merged - unless you have some remarks :)

Reply:
https://github.com/kubermatic/dashboard/issues/3391